app/src/main/java/com/geeksville/mesh/ui/contact/Contacts.kt

```diff
@@ -238,6 +247,32 @@ fun MuteNotificationsDialog(
             title = { Text(text = stringResource(R.string.mute_notifications)) },
             text = {
                 Column {
+                    // Show current mute status
+                    selectedContactKeys.forEach { contactKey ->
+                        contactSettings[contactKey]?.let { settings ->
+                            val now = System.currentTimeMillis()
+                            val statusText =
+                                when {
+                                    settings.muteUntil > 0 && settings.muteUntil != Long.MAX_VALUE -> {
+                                        val remaining = settings.muteUntil - now
+                                        if (remaining > 0) {
+                                            stringResource(R.string.mute_status_muted_for) +
```

Comment:
Might want to change this and below to formatted strings, since translations can get wonky with concatenation.

Reply:
Just realized these are plurals, too - can of worms. Looks good as-is with your changes 😅

Reply:
yeah, in is better than perfect in this case.